charts/rancher-logging/100.1.2+up3.17.4/templates/_generic_logging.yaml

```diff
@@ -0,0 +1,116 @@
+{{- define "logging-operator.logging.tpl" -}}
+apiVersion: logging.banzaicloud.io/v1beta1
+kind: Logging
+metadata:
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "logging-operator.labels" . | indent 4 }}
+spec:
+  controlNamespace: {{ .Release.Namespace }}
+  fluentbit:
+    image:
+      repository: {{ template "logging-operator.fluentbitImageRepository" . }}
+      tag: {{ template "logging-operator.fluentbitImageTag" . }}
+    {{- if not .Values.disablePvc }}
```

Comment:
Yes. We won't need to change the logic for fluentbit since it does not require `disablePvc` to be set to `true` unlike fluentd. Referred to this documentation for [fluentd](https://banzaicloud.com/docs/one-eye/logging-operator/configuration/fluentd/#custom-fluentd-hostpath-volume-for-buffers) and [fluentbit](https://banzaicloud.com/docs/one-eye/logging-operator/configuration/fluentbit/#hostpath-volumes-for-buffers-and-positions).